ci_framework/roles/build_containers/molecule/default/prepare.yml

```diff
@@ -23,3 +23,12 @@
   roles:
     - role: test_deps
     - role: repo_setup
+  tasks:
+    - name: Install jmespath package
+      become: true
+      ansible.builtin.package:
+        name: python3-jmespath
+        state: present
+
+    - name: Install community.general
+      ansible.builtin.command: ansible-galaxy collection install community.general --upgrade
```

Comment:
It is no longer needed in favor of https://github.com/openstack-k8s-operators/tcib/pull/91